### DISCUSSION TITLE
Welcome to ML-CaPsule Discussions!

### DISCUSSION THREAD

**Discussion by Niketkumardheeryan** [Niketkumardheeryan/ML-CaPsule#241] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Amarta113** [comment]
@Niketkumardheeryan, most of the issues are labeled as GSSOC22. GSSOC24 labeled issues do not exist.

**Comment by Rishi2003Das** [comment]
Please make the labels in the issues clearly like for GSSOC'24 else it's difficult to change parts in it

**Comment by Niketkumardheeryan** [comment]
done

**Comment by Calc5716** [comment]
Hi, I am Paramesh. I am a Statistics UG Student and a GSSoC '26 contributor. I already have some good ml projects and also some new ideas which I will complete and try to add it here, will start contributing from June.

Thank You
